engineering_tools/carma_script_extensions/cloud-sim

```diff
@@ -150,6 +186,14 @@ cloud-sim__start() {
         echo "Configuration does not include AWS User ARN. Please run: carma cloud-sim config -v <Desired config version>"
     fi
 
+    if [ -z "$AMI_ID" ]; then
+        echo "Configuration does not include AMI ID. Please run: carma cloud-sim config -am <AMI ID>"
+    fi
+
+    if [ -z "$REGION" ]; then
+        echo "Configuration does not include region. Please run: carma cloud-sim config -r <Region>"
```

Comment:
Since we do have a default value we use if the region is not specified, can we update this message to be something like "Configuration does not include region, defaulting to ${local_aws_region} . To change region please..."